### ISSUE TITLE
[RFE] Export folders with original names and preset image names 

### ISSUE THREAD

**Issue by AutoGitr** [WebTools-NG/WebTools-NG#681]
**Is your feature request related to a problem? Please describe.**
The program can export (backup) but when restoring using this backup the names work best if they are identical to the original folders, and assets should have the correct name.
If the program could be used to export the folder names for all libraries as they are, they would be immediately matched by any media management program when restoring a backup.
If the file names were preset, they would match Plex's local media asset naming scheme when restoring from a backup.

**Describe the solution you'd like**

New ExportTools Settings options:

1. An option to export in original folder tree, including season folders for tv libraries
2. Another option to export art and posters as background.jpg and poster.jpg
3. An option to export season posters to their season folder

**Describe alternatives you've considered**
Tautulli export, but could not get it to do what is described above.

**Additional context**
https://support.plex.tv/articles/200220677-local-media-assets-movies/

**Comment by ukdtom** [comment]
Well, when you select to export episodes, you have a setting to do almost what you want

![image](https://user-images.githubusercontent.com/4699449/222784577-14e5c2bb-a0a5-4944-9b5a-36816838d4c5.png)

Only naming is a little bit different from above req, but think you could use that regardless?

**Comment by AutoGitr** [comment]
Yes, but only almost. I do use those options currently, but the names will never match if one has a special naming scheme, which I do. With original names, it would work for everyone. And I'm guessing this wouldn't be too hard to implement, since it just gets the folder names without changing anything.

**Comment by ukdtom** [comment]
Well, currently, we don't look at the file naming at all, but use titles.

But guess it can be done

**Comment by kiwichick888** [comment]
I would also like to be able to export with folder names as per their original names.
